src/status_im/contexts/shell/activity_center/notification/syncing/view.cljs

```diff
@@ -0,0 +1,78 @@
+(ns status-im.contexts.shell.activity-center.notification.syncing.view
+  (:require
+    [quo.core :as quo]
+    quo.theme
+    [react-native.core :as rn]
+    [status-im.common.new-device-sheet.view :as new-device-sheet]
+    [status-im.contexts.shell.activity-center.notification.common.view :as common]
+    [utils.datetime :as datetime]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- swipeable
+  [{:keys [extra-fn]} child]
+  [common/swipeable
+   {:left-button    common/swipe-button-read-or-unread
+    :left-on-press  common/swipe-on-press-toggle-read
+    :right-button   common/swipe-button-delete
+    :right-on-press common/swipe-on-press-delete
+    :extra-fn       extra-fn}
+   child])
+
+(defn installation-created-view
+  [{:keys [notification extra-fn]}]
+  (let [{:keys [installationId read timestamp]} notification
```

Comment:
@Parveshdhull, notifications have a central place to transform data to idiomatic Clojure. It's error prone to mix different cases, so it's better if we stick with kebab case so that we are good Clojure citizens 😄 The function to rename `installationId` is `legacy.status-im.data-store.activities/<-rpc`.

Side note: the preferred way is to move these legacy data store files to their respective `context/` directories, as we do in `contexts/wallet/data_store.cljs` or `contexts/network/data_store.cljs`. Just mentioning since it's related, but I'm not suggesting a change for this PR :)